gcp/access_context_manager/service_perimeter_regular/locals.tf

```diff
@@ -1,11 +1,11 @@
 locals {
-  ingress_file      = try(fileexists(var.ingress_file_path), fileexists("./ingress_policies.yml")) ? file(var.ingress_file_path) : null
```

Comment:
Removing uneccesssary try function because `var.ingress_file_path` already defaults to `./ingress_policies.yml`, and if var.ingress_file_path is set it but doesn't exist, the error would probably be more relecant than defaulting to another path